packages/abstractions/kiota_abstractions/serialization/parse_node.py

```diff
@@ -118,7 +118,7 @@ def get_time_value(self) -> Optional[time]:
         pass
 
     @abstractmethod
-    def get_collection_of_primitive_values(self) -> List[T]:
+    def get_collection_of_primitive_values(self, primitive_type) -> Optional[List[T]]:
```

Comment:
If we're changing the return type to Optional, we should do so as well in the other collection methods.

Reply:
Changed this back as non of the implementations seem to return none at the moment.